sample/main.go

```diff
@@ -10,6 +10,8 @@ import (
 func main() {
 	logger := log.New(os.Stdout, "[logger] ", 0)
 
+	// domain refresh to what precedes .freshdesk.com
```

Comment:
Do we need to keep this sample file?

Reply:
It's always nice to have example to get started faster when using  a lib.